cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/AbstractConductingEquipmentConversion.java

```diff
@@ -202,16 +202,24 @@ String iidmVoltageLevelId(int n) {
     }
 
     protected VoltageLevel voltageLevel() {
-        return terminals[0].voltageLevel;
+        if (terminals[0].iidmVoltageLevelId != null) {
```

Comment:
Is `terminalData.voltageLevel` still used? If not, you can delete the field in the `TerminalData` object.

Reply:
It is used.